src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -446,7 +473,47 @@ const NetworkModificationNodeEditor = () => {
             studyUuid,
             currentTreeNode?.id,
             [...selectedItems.values()].map((item) => item.uuid)
-        ).then();
+        )
+            .then()
+            .catch((errmsg) => {
+                console.error('doDeleteModification error:', errmsg);
+                snackError('', 'errDeleteModificationMsg');
+            });
+    };
+
+    const doCopyModification = () => {
+        // just memorize the list of selected modifications
+        let newCopySet = new Set();
+        selectedItems.forEach((item) => {
+            if (item.type !== 'GROOVY_SCRIPT') {
+                newCopySet.add(item.uuid);
+            }
+        });
+        setCopiedModifications(newCopySet);
+    };
+
+    const doPasteModification = () => {
+        duplicateModifications(
+            studyUuid,
+            currentTreeNode.id,
+            Array.from(copiedModifications.keys())
```

Comment:
Array.from(copiedModifications) should work as well

Reply:
noted